docs/build.md

```diff
@@ -20,14 +20,67 @@ Set up your build chroot:
     $ export HOMEWORLD_CHROOT="$HOME/homeworld-chroot"     # this can be any directory you choose
     $ ./build-chroot/create.sh
 
-You might consider adding the variable declaration to your ~/.profile.
+You might consider adding the variable declaration to your ~/.bashrc.
```

Comment:
Not everyone uses bash.  `.profile` is the correct place for environment variables.